### PR TITLE
feat: Add a scheduled job to resubscribe the calendars

### PR DESCRIPTION
**Pull request by peterstarling**
see: https://trello.com/c/YCqoAePq/1024-add-a-cron-job-to-resubscribe-to-calendar-event-changes

### CODE REVIEW

File: apps/asap-server/src/handlers/jobs/resubscribe-calendars.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+export const resubscribeCalendarsHandlerFactory = (
+  calendarController: CalendarController,
+  unsubscribe: UnsubscribeFromEventChanges,
+  subscribe: SubscribeToEventChanges,
+): ScheduledHandlerAsync => async () => {
+  const now = DateTime.local();
+  const duration24h = Duration.fromObject({ day: 1 });
+  const calendars = await calendarController.fetchRaw({
+    maxExpiration: now.plus(duration24h).toMillis(),
+    take: 100,
```
Comment (peterstarling) [inline apps/asap-server/src/handlers/jobs/resubscribe-calendars.ts:22]: FYI i dont think we expect a 100 calendars to all expire at the same time and if they do - we will resubscribe only the first 100 then on the next day - the remaining ones

File: packages/squidex/src/squidex.ts [hunk 2]
```diff
@@ -94,6 +94,12 @@ export class Squidex<T extends { id: string; data: Record<string, unknown> }> {
         };
       }
 
+      // eslint-disable-next-line no-console
```
Comment (johnytiago) [inline packages/squidex/src/squidex.ts:97]: Isn't the logger on the http error handler not working? 
https://github.com/yldio/asap-hub/blob/master/packages/services-common/src/framework/lambda.ts#L70
Old lambdas, still run through this code. If squidex errors are not showing up even with `DEBUG=http` the issue should get fixed there.

File: serverless.js [hunk 3]
```diff
@@ -202,6 +202,25 @@ module.exports = {
         }}`,
       },
     },
+    resubscribeCalendars: {
+      handler:
```
Comment (johnytiago) [inline serverless.js:206]: Is 16 seconds of default `timeout` enough for this lambda? We have quite a few orchestrations to do. Maybe we can bump this a bit, just to be safe. What do you think?

**Comment by johnytiago** [comment]
Removing the WIP label as this no longer in progress

**Review by johnytiago — APPROVED** [review]

**Pull request merged**